src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -733,6 +731,9 @@ private boolean getJsonConfigs() {
                 lastJsonConfigTs = timestamp;
                 update = true;
                 log.info("update is in order - updating timestamp: " + lastJsonConfigTs);
+                for (String checkName : adJsonConfigs.keySet()) {
+                    log.debug("check " + checkName + " will be run");
```

Comment:
just to make the message a bit more explicit, what do you think of:
```suggestion
                    log.debug("received config for check '" + checkName + "'");
```